pkg/cri/sbserver/container_create_linux.go

```diff
@@ -106,7 +106,7 @@ func (c *criService) containerSpecOpts(config *runtime.ContainerConfig, imageCon
 		specOpts = append(specOpts, seccompSpecOpts)
 	}
 	if c.config.EnableCDI {
-		specOpts = append(specOpts, customopts.WithCDI(config.Annotations, config.CDIDevices))
+		specOpts = append(specOpts, customopts.WithCDI(config.Annotations, config.CDIDevices, config.GetMounts()))
```

Comment:
What does `config.GetMounts()` return? How do we distinguish between "mounts" (really CDI device names) that need to be handled by CDI and through regular means?

Reply:
ContainerConfig.Mounts = config.GetMounts()